.github/workflows/create_release.yaml

```diff
@@ -108,13 +108,13 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - uses: actions/setup-dotnet@v3.0.3
-        with: 
+        with:
           dotnet-version: ${{ env.DOTNET_VERSION }}
       - name: Package Azure Function
         run: |
           dotnet publish LoRaEngine/LoraKeysManagerFacade -c Release
           FUNCTION_ZIP=function-${VERSION}.zip
-          zip -r $FUNCTION_ZIP LoRaEngine/LoraKeysManagerFacade/bin/Release/net${DOTNET_VERSION%??}
+          cd LoRaEngine/LoraKeysManagerFacade/bin/Release/net${DOTNET_VERSION%??} && zip -r $FUNCTION_ZIP . && mv $FUNCTION_ZIP ../../../../..
```

Comment:
Rather than moving the function zip, couldn't we just properly set the environment variable on row 119 instead? Something like: FUNCTION_ZIP=$(pwd)/$FUNCTION_ZIP could work?

Reply:
sure

Reply:
fixed in https://github.com/Azure/iotedge-lorawan-starterkit/actions/runs/3558870073